ludwig/schema/combiners/tabnet.py

```diff
@@ -46,7 +46,7 @@ class TabNetCombinerConfig(BaseCombinerConfig):
     )
 
     bn_momentum: float = schema_utils.FloatRange(
-        default=0.95, description="Momentum of the batch norm. `m_B` in the paper."
+        default=0.05, description="Momentum of the batch norm. `m_B` in the paper."
```

Comment:
I would update the description to reflect the "inversion" otherwise someone looking at the paper may be confused